### ISSUE TITLE
Tutorial layout - vertical space

### ISSUE THREAD

**Issue by Jaqster** [microsoft/pxt-microbit#5064]
I am concerned about the vertical space required for this new tutorial layout.  Especially for tutorials with multiple lines, students must scroll down the instructions in order to get to the Next button.  We should test on small screens.

![image](https://user-images.githubusercontent.com/22108954/233701931-d2aae0da-7f74-47f8-b8e4-7f6b2338decf.png)

@microbit-giles, @microbit-lucy - would be great to get your input on the new tutorial layout in https://makecode.microbit.org/beta

**Comment by abchatra** [comment]
I am bit concerned that next and bubbles are hidden too. Is this by design?

**Comment by thsparks** [comment]
Yes the hidden bubbles is by design. Apparently when they were always visible students would just click Next without scrolling.

**Comment by jwunderl** [comment]
for what it's worth, if I recall correctly that's why the design pushed the next button to the bottom (and also to make it scroll enough), but the bubbles were fixed at the top so they were always visible / a back up

**Comment by microbit-giles** [comment]
The layout looks good to me. The vertical space seems ok as we would not include 4 lines of title, blank lines and intro text as in the example above, we would typically include just 1 or 2 lines of direct instruction in each step.

**Comment by thsparks** [comment]
@Jaqster Any reason to keep this issue open, or can we close it now? I'm not currently aware of any action needed, but could be missing something.
